facia-press/app/frontpress/PressedCollectionDeduplication.scala

```diff
@@ -42,11 +42,15 @@ object PressedCollectionDeduplication {
     pCVs.flatMap{ collection => (collection.pressedCollection.curated.take(visibility) ++ collection.pressedCollection.backfill.take(visibility)).map ( pressedContent => pressedContent.header.url ) }
   }
 
+  def makeNewBackfill(collectionV: PressedCollectionVisibility, preceedingCollectionVsDeduplicated: Seq[PressedCollectionVisibility]): List[PressedContent] = {
```

Comment:
What do you think about naming this something like `removeDuplicatesFromBackfill`, just to be super clear what the intent is here?

Reply:
+1